scripts/convert_lit_checkpoint.py

```diff
@@ -103,10 +103,21 @@ def copy_weights_gpt_neox(
 @torch.inference_mode()
 def convert_lit_checkpoint(
     *,
-    checkpoint_name: str,
+    checkpoint_name: str = "",
     checkpoint_dir: Path = Path("checkpoints/tiiuae/falcon-7b"),
     model_name: Optional[str] = None,
 ) -> None:
+    # pydantic or jsonargparse do not have a clear way to allow for
```

Comment:
You can simply do `CLI(as_positional=False)`

Reply:
updated in [872606e](https://github.com/Lightning-AI/lit-gpt/pull/321/commits/872606e72a222cc8c1c039c41819879983c64c29)